resources/lang/en/patrols/new_cat.json

```
@@ -4327,5 +4327,393 @@
                 "outsider_rep": -2
             }
         ]
+    },
+    {
+        "patrol_id": "gen_train_app_newfriend1",
+        "biome": ["any"],
+        "season": ["any"],
+        "types": ["training"],
+        "tags": ["new_cat"],
+        "patrol_art": "train_general_intro",
+        "min_cats": 1,
+        "max_cats": 1,
+        "min_max_status": {
+            "apprentice": [1, 1]
+        },
+        "weight": 20,
+        "intro_text": "p_l is out doing some solo training when, suddenly, {PRONOUN/p_l/subject} {VERB/p_l/spot/spots} another cat around {PRONOUN/p_l/poss} age.",
+        "decline_text": "{PRONOUN/p_l/subject/CAP} {VERB/p_l/elect/elects} to ignore the stranger and focus on {PRONOUN/p_l/poss} training.",
+        "chance_of_success": 50,
+        "success_outcomes": [
+            {
+                "text": "A loner! p_l knows that {PRONOUN/p_l/subject} should head back to camp and report this, but...{PRONOUN/p_l/subject} can't help but be a little curious. p_l approaches the loner and strikes up a conversation and, without either cat realizing it, they wind up talking and training together for hours.",
```

```suggestion
                "text": "A loner! p_l knows that {PRONOUN/p_l/subject} should head back to camp and report this, but... {PRONOUN/p_l/subject} can't help but be a little curious. p_l approaches the loner and strikes up a conversation. Without either cat realizing it, they wind up talking and training together for hours.",
```

Missing a space, fixed run-on sentence.